### PR TITLE
[DO NOT MERGE] Test updating classic dark default theme to use "blueberry blue" colors

### PR DESCRIPTION
**Pull request by agrullon95**
Related to https://github.com/Automattic/dotcom-forge/issues/4615

## Proposed Changes

* Testing PR for pink to "blueberry blue" color changes. 
Partially updates the classic-dark theme to match the color scheme and then updates some of the onboarding flows to use blueberry. 

Work in progress for follow up: Other areas such as input fields, hover/focus colors, site icon picker, etc. should ideally be updated as well

## Testing Instructions

* Checkout branch or use live link
* Navigate through main onboarding flows and look at CTA color changes. 
such as `/setup/blog`, `/setup/newsletter`, `/setup/link-in-bio`,
* Navigate through `/log-in`
* Navigate through sign up `/start` then `Continue with Email`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by lsl** [comment]
Not sure about the approach, lets keep discussing on slack.

I don't want this to turn into a huge audit of all the overrides but these are ones I see quickly that might be visible enough to fix up front.

Menus: Selected menu color + hover text color + comment count color
Page: link color should probably update to blueberry?
Zendesk chat color: might not be able to change this but if we can that would be nice to move to blueberry too

Screenshot_2023-12-14_09-23-05

Not sure how to fix this one (if you even should?), might need to change colors for personal, premium and business all at once:
Screenshot 2023-12-14 at 09-23-36 Plans ‹ test — WordPress com

File: client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss [hunk 1]
```diff
@@ -104,14 +104,14 @@
...
 			&:focus {
 				box-shadow: none;
-				outline: 2px solid var(--studio-blue-60);
+				outline: 2px solid var(--color-accent-60);
 				outline-offset: 1px;
-				background: var(--studio-blue-60);
+				background: var(--color-accent-60);
 			}
 			&:hover {
 				background-color: var(--wp-admin-theme-color-darker-10);
```
Comment (lsl) [inline client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss]: Will want a --color-accent-60 here too, (I just replaced it with blue 60 a few minutes ago https://github.com/Automattic/wp-calypso/pull/85337)

File: client/my-sites/plugins/education-footer/index.tsx [hunk 2]
```diff
@@ -210,7 +210,7 @@ function ReadMoreLink() {
 	const { __ } = useI18n();
 
 	return (
-		<CardText color="var(--studio-blue-50)">
+		<CardText color="#3858e9">
```
Comment (lsl) [inline client/my-sites/plugins/education-footer/index.tsx:213]: Is there somewhere we can put these as variables?

**Comment by lsl** [comment]
@agrullon95 could you please pick these blues up too? (either this PR or a follow up :pray: ) Looks like there are a couple more usages of the same hexcode listed under something called "richblue".

https://github.com/Automattic/wp-calypso/blob/59e6b2646cd87da88de99576b35b9ae6971e6b9b/client/login/magic-login/style.scss#L207-L209

Seen here:

https://github.com/Automattic/wp-calypso/assets/811776/c2aaa02e-6e97-4233-b8bb-00ae288cc53f

File: client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss (hunk 1 above)
Comment (agrullon95) [reply client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss]: Updated

File: client/my-sites/plugins/education-footer/index.tsx (hunk 2 above)
Comment (agrullon95) [reply client/my-sites/plugins/education-footer/index.tsx:213]: Hmmm i can look into this , possibly follow up PR

**Comment by agrullon95** [comment]
>@agrullon95 could you please pick these blues up too? (either this PR or a follow up 🙏 ) Looks like there are a couple more usages of the same hexcode listed under something called "richblue".

@lsl - I updated the styling for the magic login button and added a hover state since I noticed it was missing on prod.
I think the richblue is fine as is, but i'll look a bit more into it. Thanks for the reviews thus far.

**Comment by lsl** [comment]
No problem, I'm just glad someone is able to dedicate some time for this.

Have you checked the editor? :smile: 

I'm assuming this is done via etk, but I couldn't get it to apply the new theme using:

```
install-plugin.sh editing-toolkit update/default-classic-dark-theme-pink-color-scheme-v2
```

Maybe I'm not sandboxing the right urls?

**Pull request closed without merging**